src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -84,6 +87,26 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
             </>
           )}
         </button>
+        {!isExpanded && (
+          <button
+            aria-label={isExpanded ? "Hide chat" : "Show chat"}
+            className="chat-sidebar__controller"
+            onClick={togglePrivateChatSidebar}
+            style={{ marginTop: 55 }}
```

Comment:
Style property values should be part of the SCSS classes i.e. no inline styles unless there's no way around it.